airflow/dags/parse_rt/create_service_alerts_external_table.yml

```diff
@@ -0,0 +1,134 @@
+operator: operators.ExternalTable
+dependencies:
+  - parse_rt_service_alerts
+source_objects:
+  - "service_alerts/*.jsonl.gz"
+destination_project_dataset_table: "gtfs_rt.external_service_alerts"
+source_format: NEWLINE_DELIMITED_JSON
+use_bq_client: true
+hive_options:
+  mode: AUTO
+  source_uri_prefix: "/service_alerts/"
+schema_fields:
+  - name: metadata
```

Comment:
Yes, @edasmalchi we can chat at some point too. I'm OK with this being a breaking change because we'll want queries to start using partition columns anyways.

Reply:
Thanks for the heads up! You can go ahead and break it -- I've ran all the queries I need to for the current iteration of the speedmap site and clearly partitions will help things a bunch. Would of course appreciate a walkthrough after the changes are done